src/PKSim.Core/Services/IChartTask.cs

```diff
@@ -0,0 +1,36 @@
+﻿using OSPSuite.Core.Chart;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using PKSim.Core.Model;
+
+namespace PKSim.Core.Services
+{
+   public interface IChartTask
+   {
+      /// <summary>
+      /// updates all observed data defined in the simulation to the underlying charts of this simulation
+      /// </summary>
+      /// <param name="simulation">Simulation used to retrieve the charts to update</param>
+      /// <param name="project">Optional project. If undefined, it will be retrieved from the project retriever</param>
+      void UpdateObservedDataInChartsFor(Simulation simulation, IProject project = null);
```

Comment:
I moved this baby to core (where it should have been in the first place) so that I could use it from snapshot